src/components/pages/HomePage/time.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Inspired by James Tauber's Mars Clock
+ * https://github.com/jtauber/mars-clock
+ */
+
+// Difference between TAI and UTC. This value should be
+// updated each time the IERS announces a leap second.
+var tai_offset = 37;
+
+// Last time the above value changed.
+var last_leap_second = "1 January 2017";

```

Comment:
Vars are genereally not used nowadays. These here can be Const, since they are not changed within the code

Reply:
"generally not used" is somewhat of an overstatement.
The scope of a var is not so often needed in React.